libsplinter/src/auth/oauth/error.rs

```diff
@@ -17,6 +17,28 @@
 use std::error::Error;
 use std::fmt;
 
+/// An unrecoverable error that can occur when using an OAuth client
+#[derive(Debug)]
+pub struct OAuthClientError(pub String);
```

Comment:
This form isn't consistent with our recent discussions that stick this in a "message" field.

Reply:
Updated to use a `message` field.